types.ts

```diff
@@ -1351,6 +1351,13 @@ export interface Skill {
   tools: Tool[];
 }
 
+export interface SkillEntrypoints {
+  /** Skill to be invoked when the agent is clicked on in the bench for the first time. */
+  benchInitialization?: {
+    skillName: string
+  };
```

Comment:
I think the next time you add an entrypoint you'll want to pull out this object shape into something named to reuse, but can wait until that's needed.